### ISSUE TITLE
ctrl-space shortcut not working in QS 2.0

### ISSUE THREAD

**Issue by eisGeist** [quicksilver/Quicksilver#2694]
Dear developers,

I am submitting this to let you know about a thing that, to me, appear as a bug.
If I am missing the point, please head me to the right place where I can find more information.

Yesterday, I updated to QS 2.0 and realised the shortcut ctrl-space was not working.

I headed to the preferences, command section, and saw it was set to ctrl-1.
I tried to revert it to ctrl-space, but when hitting ctrl-space to set it as my choice the text-box kept displaying ctrl-1.

I had to adapt to trigger QS by hitting twice the ctrl key.

This morning I downgraded it to 1.6.1.

Many thanks in advance for your attention!

With best regards,

-- 
lS.

**Comment by skurfer** [comment]
Are you sure you have the official build? I can’t reproduce this problem. See #2682

If it still won’t work, you can set the shortcut using version 1.6.1, then upgrade, or since what you want to use is Quicksilver’s default, you can run this in Terminal:

```
defaults delete com.blacktree.Quicksilver QSActivationHotKey
```

**Comment by eisGeist** [comment]
Dear Rob,

thank you for your reply and for pointing me to the issue [#2682](https://github.com/quicksilver/Quicksilver/issues/2682)

I updated by clicking on "Install Update" when prompted by QS, so I guess I used the official build.
After receiving your reply I decided to give v 2.0 another go. 
Result:
The "task viewer" floats on my computer monitor showing once in a while "Catalog updating" and there was no way to close it without quitting QS. 
I also downloaded the DMG file from your website: same issues.

Downgraded again to 1.6.1

I'll keep eye on the issue [#2682](https://github.com/quicksilver/Quicksilver/issues/2682), when it will be solved I upgrade to QS 2.*.

Thank you again!

**Comment by gabrielparriaux** [comment]
Dear @skurfer, 

I have the same problem as described by @eisGeist. I upgraded QS from inside the app to the latest official release — v2.0.0 (4027). I used the default shortcut (ctrl + spacebar) and it got changed to ctrl + 1. In the settings, it’s impossible to enter the spacebar (any other key is working). I tried to use the terminal command you mention in your message (having first quit QS), but nothing changes. Default reverses to ctrl + 1. 

I’m not sure to understand if it’s necessary for me to install again the old 1.6.1 version, set the shortcut, and upgrade again, as this would result in exactly the same situation that I had before (the correct shortcut in 1.6.1 and an upgrade to version 2).

Thanks a lot for your help,

Gabriel

**Comment by gabrielparriaux** [comment]
My system is in French and Keyboard is Swiss-French (QWERTZ), don’t know if it can be relevant…

**Comment by natcl** [comment]
Interesting, French Canadian keyboard over here.

**Comment by n8henrie** [comment]
FWIW I haven't been able to reproduce.

My default is <kbd>^</kbd><kbd>space</kbd>, which has been working throughout 2.0 development and currently on the release.

I downgraded to 1.6.1 and then went through the in-app upgrade back to 2.0 and it continues to work.

I reset the shortcut to <kbd>^</kbd><kbd>9</kbd> and it works. I set it back to <kbd>^</kbd><kbd>space</kbd> and it works.

¯\\\_(ツ)_/¯

**Comment by natcl** [comment]
Is it because we use cmd-space?

**Comment by natcl** [comment]
Is it because we use cmd-space?

**Comment by natcl** [comment]
Is it because we use cmd-space?

**Comment by natcl** [comment]
Is it because we use cmd-space?

**Comment by natcl** [comment]
I use cmd-space cloud that be an issue ?

**Comment by aspacca** [comment]
setting to `option+space` (`option+1` appears as value)
```
$ defaults read com.blacktree.Quicksilver QSActivationHotKey
{
    keyCode = 49; 
    modifiers = 524576;
}
```

setting to `option+1` (`option+1` appears as value)
```
$ defaults read com.blacktree.Quicksilver QSActivationHotKey
{
    keyCode = 18;
    modifiers = 524576;
}
```

In both cases `option+1` is the shortcut that actually works

**Comment by aspacca** [comment]
Italian keyboard mapping, not producibile on ABC one

**Comment by aspacca** [comment]
If I switch to ABC keyboard mapping on the same mac I'm able to set the shortcut to `ctrl+space` and it works properly, switching back to Italian breaks the shortcut, while in the preferences is still showed properly

**Comment by pjrobertson** [comment]
Hi all, this is a known issue with Quicksilver v2.0, and we've already issued a fix.

A duplicate issue here: https://github.com/quicksilver/Quicksilver/issues/2692 has more information.

You can manually download an updated version of the app with the fix here: https://github.com/quicksilver/Quicksilver/suites/5837416196/artifacts/196109949

@skurfer  - I suggest we release QS 2.0.1 with this in there shortly.

**Comment by n8henrie** [comment]
> I suggest we release QS 2.0.1 with this in there shortly.

I can work on a changelog -- it looks like the only merged commit in master is 3518a95df830e8d11cce9022618c8db835641034. 

What about <https://github.com/quicksilver/Quicksilver/issues/2681#issuecomment-1081333139> -- did I miss a PR somewhere that should be included?

**Comment by pjrobertson** [comment]
See my recent email.  I've just created milestone 2.1.0 (which I think should include everything we should get merged asap.)

For #2681 - the 'fix' was created in PR #1497 9 years ago. I'll refactor and make a new PR. Agreed it should go in 2.1.0 (what I'm calling your 2.0.1)
